src/endpoints/project_notes.yaml

```diff
@@ -106,7 +106,7 @@ paths:
       requestBody:
         description: JSON Patch document as defined by [RFC 6902](https://www.rfc-editor.org/rfc/rfc6902)
```

Comment:
d'oh ... change `requestBody` to reference [components/requests.yaml#/jsonPatch](https://github.com/AWeber-Imbi/imbi-openapi/blob/3292150db38f81284a107de4c871da8e77776dc0/src/components/requests.yaml#L1-L15) instead.
```yaml
      requestBody:
        $ref: '../components/requests.yaml#/jsonPatch'
```

Reply:
Ah. Updated

Reply:
bah ... i typoed :/

Reply:
nvm ... you typoed